src/lib/serviceRoutines/postContextEntitiesByEntityId.cpp

```diff
@@ -51,12 +54,11 @@ std::string postContextEntitiesByEntityId
 )
 {
   std::string  entityId = (compV[0] == "v1")? compV[3] : compV[2];
```

Comment:
I understand that we are checking in some other place that, in the case of "v1" the next token is "registry" (my concernt is due to looking to this line, it will also work with e.g. /v1/foo or /v1/bar and it shouldn't).